### PR TITLE
fixed footer position

### LINKED ISSUE #1: Footer not sticking to bottom
### Subject of the issue
When the content is less, the background shrinks and the footer does not stay at the bottom of the website.

### Environment
* Windows 10
* Chrome 90, Firefox 88

### Steps to reproduce
1. Visit http://vipsace.org/events/
2. It happens on every page where the content is less.

### Expected behavior
Footer should stick to the bottom of the website regardless of the content size.

### Actual behavior
image

### Possible solutions
1. `min-height: 100vh` on the body might do the trick.
2. We can make the content a flex box with direction column and `flex-grow: 1`. So it is always occupying the maximum space.

### PR DESCRIPTION
**Pull request by NikhilSharma03**
This PR fixes #1 

## Desciption
The footer was not behaving as it should be.

## Changes 
changes made into main.css file to get the desired output.

### CODE REVIEW

**Comment by ansh-saini** [comment]
Hey @NikhilSharma03. I'll review your PR on the weekend.
I noticed that the formatting of your PR description is not right. Please take a look at [this link](https://www.markdownguide.org/basic-syntax/) to learn more about markdown formatting and how to make your PR look beautiful.
Also, if you want to link your PR to the issue that you solved, you can mention the `#` of the issue in your PR description. The most common practice is to write `Fixes #1` on the top of your PR description. This way you won't have to go the the issue and write a comment asking the maintainers to review your Pull Request.

**Comment by NikhilSharma03** [comment]
Ok, thank you. I will keep that in mind next time.

File: website/static/website/css/main.css [hunk 1]
```diff
@@ -9915,7 +9915,10 @@ iframe {
 .wrapper-hidden {
   -webkit-transition: all 1.2s ease-in-out;
   transition: all 1.2s ease-in-out;
-  opacity: 0.5; }
+  opacity: 0.5; 
+  display: flex;
+  flex-direction: column;
+  height: 100vh; }
```
Comment (ansh-saini) [inline website/static/website/css/main.css:9921]: If the content is greater than `100vh`, this property will break the scrolling of the content.
Visit `http://localhost:8000/projects/` and try to scroll. The same is happening with mobile devices. Make sure your changes don't break existing functionality and are fully responsive across all screen sizes.

**Review by ansh-saini — CHANGES_REQUESTED** [review]

**Comment by NikhilSharma03** [comment]
@ansh-saini Can you please review?

**Pull request merged**